doc: guide for static PVC creation (#4997)
ceph/ceph-csi: docs/static-pvc.md

Now scale down the application pod which is using `cephfs-static-pvc` and scale
up the application pod to resize the filesystem.

> [!note]
> If you have mounted same static PVC to multiple application pods, make
sure you will scale down all the application pods and make sure no application
pods using the static PVC is running on the node and scale up all the
application pods again(this will trigger `NodeStageVolumeRequest` which will
resize the filesystem for static volume).

> ### Verify RBD static PVC creation

Reviewer: ```suggestion
### Verify RBD static PVC
```

> We can verify the RBD static PVC creation by running a Pod that mounts the PVC.

Reviewer: ```suggestion
We can verify the RBD static PVC by creating a Pod that mounts the pvc.
```

```yaml
apiVersion: v1
kind: Pod
metadata:
  name: rbd-static-pvc-test
  namespace: default
spec:
  containers:
    - name: busybox
      image: busybox:latest
      volumeMounts:
        - name: static-pvc
          mountPath: /data/pvc
      command: ["sleep", "3600"]
  volumes:
    - name: static-pvc
      persistentVolumeClaim:
        claimName: fs-static-pvc
```

```bash
$ kubectl create -f rbd-static-pvc-test.yaml
pod/rbd-static-pvc-test created
```

Verify that the PVC has been successfully mounted.

```bash
$ kubectl exec rbd-static-pvc-test -- df -h /data/pvc
```

> After completing the verification steps, you can delete the test Pods for RBD:

Reviewer: ```suggestion
Once you have completed the verification step, you can delete the test Pod for RBD.
```

  volumeMode: Filesystem
```

> If you are a [Rook](https://rook.io/) user, there are some differences, such as:

Reviewer 2: The above example should be enough. Let's not have Rook specific example here and anyway there isn't any extra configuration needed for Rook.

persistentvolumeclaim/cephfs-static-pvc created
```

> ### Verify PVC creation

Reviewer: can you please update the rbd example with the verification step as well?

Author: Sure, but in our scenario, we use OpenEBS to provide LocalPV for database service persistence, so I cannot provide a solution immediately. 

I will test it in the cluster and provide a verification method suitable for RBD🫡.

persistentvolumeclaim/cephfs-static-pvc created
```

> ### Verify CephFS static PVC creation

Reviewer: ```suggestion
### Verify CephFS static PVC
```

> We can verify the CephFS static PVC creation by running a Pod that mounts the PVC.

Reviewer: ```suggestion
We can verify the CephFS static PVC by creating a Pod that mounts the pvc.
```
